neqo-server/src/main.rs

```diff
@@ -318,9 +318,15 @@ impl QuicParameters {
 }
 
 fn emit_packet(socket: &mut UdpSocket, out_dgram: Datagram) {
-    let sent = socket
-        .send_to(&out_dgram, &out_dgram.destination())
-        .expect("Error sending datagram");
+    let sent = match socket.send_to(&out_dgram, &out_dgram.destination()) {
+        Err(ref err) => {
+            if err.kind() != io::ErrorKind::WouldBlock {
```

Comment:
Do you want to avoid logging Interrupted as well?

Reply:
A random thought:

```rust
pub trait IsRetryable {
  fn is_retryable(&self) -> bool;
}

impl IsRetryable for std::io::Error {
  fn is_retryable(&self) -> bool {
    e.kind() == ErrorKind::WouldBlock || e.kind() == ErrorKind::Interrupted
  }
}
```

Reply:
I might add this as part of the addition of socket I/O to neqo-common. Right now I'd need to duplicate it in the client and the server.